server/spec/cert_revocation_spec.rb

```diff
@@ -155,11 +155,11 @@
     certs.length.should == 1
 
     cert_serial = certs[0].serial.serial
-    expect(revoked_serials).to_not include(cert_serial)
+    revoked_serials.should_not include(cert_serial)
 
     @cp.update_owner(cam_owner['key'], {'contentAccessMode' => "entitlement"})
     new_system.list_certificates.length.should == 0
-    expect(revoked_serials).to include(cert_serial)
+    revoked_serials.should include(cert_serial)
```

Comment:
The ```expect().[not_]to``` testing is what we should be using in rspec. ```.should``` is the depreciated one.

Reply:
I realized after posting this that it was likely automatically done as part of a revert command. That said, if we end up keeping any component of the original work, we should also keep these. If this ends up being just a full reversion, then the above comment can be ignored.

Reply:
This was done as a part of the revert command. Now updated it manually to use `expect()` in the spec.

Reply:
:+1:
Thanks!